repos/tap/utils/definitions/copyToDefinitionClipboard.js

```diff
@@ -0,0 +1,23 @@
+import { Clipboard } from 'react-native'
```

Comment:
FYI this is deprecated in react-native.

They recommend using a community version.

We should probably add our own to keg-components or something, especially since I'm not sure those community variants implement a web version.

Reply:
Word, I did not know that. Thanks for the heads up. I'll leave it for now and we can update down the road
This is just a place-holder anyways. I'd to have it auto-add the defintion to the feature editor, but this was the quickest solution given the time and that it's an MVP